### PR TITLE
Add docs for new official Neo4j image

### PR DESCRIPTION
**Pull request by benbc**
These are docs changes corresponding to https://github.com/docker-library/official-images/pull/1096.

### CODE REVIEW

File: neo4j/user-feedback.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Documentation
+
+Documentation for this image is stored in the[`%%REPO%%/` directory](https://github.com/docker-library/docs/tree/master/%%REPO%%) of the[`docker-library/docs` GitHub repo](https://github.com/docker-library/docs). Be sure to familiarize yourself with the[repository's `README.md` file](https://github.com/docker-library/docs/blob/master/README.md) before attempting a pull request.
```
Comment (psftw) [inline neo4j/user-feedback.md]: hmm missing spaces preceding a number of links in this file

**Comment by psftw** [comment]
Nice work!
I think the CI is mad about seeing changes to README.md, so removing that from the PR might do the trick.  nitpick: ``neo4j/neo4j`` -> ``neo4j`` :-)

File: neo4j/user-feedback.md (hunk 1 above)
Comment (benbc) [inline neo4j/user-feedback.md]: Weird. Fixed. Thanks.

**Comment by benbc** [comment]
Oh. I misunderstood https://github.com/docker-library/docs/#image-namereadmemd to mean that *I* should generate the file myself. I've removed it.

**Comment by benbc** [comment]
fixed the references to `neo4j/neo4j`

**Comment by tianon** [comment]
LGTM

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**